assets/cubyz/biomes/ocean/base.zig.zon

```diff
@@ -38,14 +37,9 @@
 	},
 	.transitionBiomes = .{
 		.{
-			.id = "cubyz:beach",
-			.chance = 1,
-			.width = 1,
-			.properties = .{.land, .inland},
-		},
-		.{
-			.id = "cubyz:ocean_shelf",
```

Comment:
Could your explain your reasoning for removing the ocean shelf? It does seem to make the transition a lot more extreme, Removing it seems to make the transition pretty harsh in some cases:
![Screenshot at 2025-01-26 15-58-27](https://github.com/user-attachments/assets/7b915f9f-ecd0-4e59-bb10-64a4a9aea815)
keepOriginalTerrain seems to help a little bit in some cases, but it doesn't appear to reliably solve the issue.

Reply:
It made the transition smoother by introducing an extra biome slightly below the water surface.